lib/vanagon/platform.rb

```diff
@@ -435,6 +435,31 @@ def add_build_repository(*args)
       end
     end
 
+    # Save the generic compiled archive and relevant metadata as packaging
+    # output. This will include a json file with all of the components/versions
+    # that were built and a bill of materials when relevant. The archive will be
+    # a gzipped tarball.
+    #
+    # @param project The Vanagon::Project to run this on
```

Comment:
probably want a `@return` saying that this returns an array of commands, rather than executing the commands themselves.

Reply:
added